routing/src/frrmi.rs

```diff
@@ -45,6 +46,20 @@ pub(crate) struct Frrmi {
     inservice: Option<FrrmiRequest>,  /* the request currently being serviced */
     timeout: Option<Instant>,         /* timeout for the current request in service */
     requests: VecDeque<FrrmiRequest>, /* queue of other requests to frr-agent */
+    stats: FrrmiStats,                /* stats */
+}
+
+/// Stats for the `Frrmi`
+#[derive(Default)]
+pub(crate) struct FrrmiStats {
```

Comment:
Just asking, are these stats exported to Prometheus or are they simply exposed - as I believe - to the CLI?